modules/monitoring-accessing-third-party-uis-using-the-web-console.adoc

```diff
@@ -0,0 +1,49 @@
+// Module included in the following assemblies:
+//
+// * monitoring/accessing-third-party-uis.adoc
+
+[id="accessing-third-party-uis-using-the-web-console_{context}"]
+= Accessing third-party monitoring UIs by using the web console
+
+You can access Prometheus, Alertmanager, and Grafana web UIs through the {product-title} web console.
+
+.Prerequisites
+
+* You have access to the cluster as a user with the `cluster-admin` role.
+
+.Procedure
+
+. Select the *Administrator perspective*.
++
+[NOTE]
+====
+Access to the third-party Prometheus, Alertmanager and Grafana UIs is not available from the Developer perspective.
+====
+
+. Access a third-party monitoring UI:
+
+* To access the Prometheus UI:
+.. Navigate to *Monitoring* -> *Metrics*.
++
+.. Select *Prometheus UI* to open the login page for the Prometheus UI.
++
+.. Select *Log in with OpenShift* to log in using your OpenShift credentials.
+
+* To access the Alertmanager UI:
+.. Navigate to the *Monitoring* -> *Alerting* page.
++
+.. Select *Alertmanager UI* to open the login page for the Alertmanager UI.
++
+.. Select *Log in with OpenShift* to log in using your OpenShift credentials.
+
+* To access the Grafana UI:
+.. Navigate to the *Monitoring* -> *Dashboards* page.
++
+.. Select *Grafana UI* to open the login page for the Grafana UI.
++
+.. Select *Log in with OpenShift* to log in using your OpenShift credentials.
+
+[NOTE]
+====
+The Thanos Querier's UI cannot be accessed from the {product-title} web console. You must obtain the Thanos Querier's UI's URL by using the CLI.
```

Comment:
all the routes could be get via `Networking -> Routes` as a user with the `cluster-admin` role

Reply:
Thank you, this is a good point. I have now removed that note and simplified the procedure in light of your comment.